docker-compose.yml

```diff
@@ -8,11 +8,44 @@ services:
     ports:
     - "5087:5000"
     volumes:
-        - ./output:/output:rw
-        - ./logs/:/app/logs
+        - ./temp:/app/temp:rw
+        - ./logs/:/app/logs:rw
     networks:
       - default
     restart: on-failure
     command: /app/run_dev_server.sh
 
+  metabolomicsusi-worker:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: metabolomicsusi-worker
+    volumes:
+      - ./temp:/app/temp:rw
+      - ./logs:/app/logs:rw
+    command: /app/run_worker.sh
+    restart: unless-stopped
+    depends_on: 
+      - metabolomicsusi-redis
```

Comment:
Do we need both RabbitMQ and Redis? Don't they do the same thing?

Reply:
So broadly the same thing but the scale at which its done is different. rabbitmq presents problems when the number of results gets large, but redis should be fine I think. The only advantage of rabbitmq is on restart to preserve the queue